src/steamship/plugin/blockifier/transcriber.py

```diff
@@ -0,0 +1,77 @@
+from abc import abstractmethod
+from typing import List, Optional, Set
+
+from steamship import Block, File, SteamshipError, Tag, Task, TaskState
+from steamship.base.mime_types import MimeTypes
+from steamship.invocable import InvocableResponse
+from steamship.invocable.plugin_service import PluginRequest
+from steamship.plugin.blockifier.blockifier import Blockifier
+from steamship.plugin.inputs.raw_data_plugin_input import RawDataPluginInput
+from steamship.plugin.outputs.block_and_tag_plugin_output import BlockAndTagPluginOutput
+
+TRANSCRIPT_ID = "transcript_id"
+
+
+class Transcriber(Blockifier):
+    @abstractmethod
+    def start_transcription(
+        self, audio_file: PluginRequest[RawDataPluginInput], mime_type: MimeTypes
+    ) -> str:
+        """Transcribe an audio file and turn it into a transcription and optional Tags."""
```

Comment:
Looks from below like the `str` return is the `transcript_id`, may want to make a note of that in the docstring since it kinda implies the method returns a transcription and tags.

Reply:
Oops. Too fast! Good catch!